src/libs/queries/sdk/strategy.ts

```diff
@@ -171,14 +162,9 @@ export const useGetUserStrategies = ({ user }: Props) => {
         getTokenById,
         importToken,
         Token,
-        roiData: roiQuery.data || [],
       });
     },
-    enabled:
-      tokens.length > 0 &&
-      isInitialized &&
-      roiQuery.isFetched &&
-      ensAddress.isFetched,
+    enabled: tokens.length > 0 && isInitialized && ensAddress.isFetched,
```

Comment:
do you want to wait for trade count to be fetched here? Would trade count not show 0 until it is loaded this way?

Reply:
I think it's better to display 0 quickly and update it after than waiting for the tradecount to be ready to display the whole page.

Reply:
No worries, if you prefer, sounds good 👍